public/index.css

```diff
@@ -329,34 +329,11 @@ code {
 	unicode-bidi: plaintext;
 }
 
-/*RTL Fix codeBlock base in Shiki output*/
-[dir='rtl'] span.line {
-	direction: ltr;
-	unicode-bidi: isolate;
-}
-
 /*RTL Fix Arrows*/
 [dir='rtl'] .toc-mobile-container svg {
 	transform: rotateY(180deg);
 }
 
-table,
```

Comment:
Looks like this style also applied to `<table>`s:

| Live site | This PR |
|---|---|
| ![Table displayed in Astro’s docs](https://github.com/withastro/docs/assets/357379/9a4a7e42-6ef5-400f-a7e1-2ddcae10bc7f) | ![Table displayed in Astro’s docs](https://github.com/withastro/docs/assets/357379/cbff5c0b-ebee-4bc0-b37a-ba752aeb64a7) |

Not sure how much we care — the tables are rarely used — but we should probably only remove `pre` from this block unless we feel like investigating thoroughly how much of these styles are needed for tables?

Reply:
First of all, thank you for looking through this so thoroughly!

The removal of `table` was actually intentional. I did check all occurrences in our live content (which are limited to exactly two tables - the one you've screenshotted, plus one in the legacy v1 migration guide), and the tables honestly looked better and more readable to me both on desktop and mobile in my new version without these styles.

Besides that, I also don't think that applying a default `pre`-like styling to every text in a table is a good idea. Tabular data isn't always automatically code. In the screenshot you provided, I'd actually find a `pre` styling of the "collection" column misleading as it distracts from the actual keywords on the right that authors may want to copy & paste into their `package.json` files.

Let me know what you think!

Reply:
Thanks for explaining. I never loved these styles (and usually advocate against table usage when it comes up 😄) so as long as this was intentional, I’m OK with it 👍